dev/release/verify-release-candidate.sh

```diff
@@ -224,6 +224,8 @@ setup_miniconda() {
     else
         MINICONDA_URL=https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
     fi
+  elif [ "$(uname)" == "Linux" ] && [ "$(uname -m)" == "aarch64" ]; then
+    MINICONDA_URL=https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-aarch64.sh
   else
     MINICONDA_URL=https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
```

Comment:
Shall we use miniforge form amd64 as well?

Reply:
Using miniforge for all platforms looks better to me.
Maybe after 4.0 release? I don't know.